modules/multi-arch-enabling-or-disabling-multiarch-operator-using-labels.adoc

```diff
@@ -0,0 +1,35 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-or-disabling-multiarch-operator-using-labels_{context}"]
+
+= Enabling or disabling the Multiarch Tuning Operator by removing the labels in a namespace
+
+After creating a `PodPlacementConfig` custom resource (CR), you can manually enable or disable the Multiarch Tuning Operator in a namespace by removing the `multiarch.openshift.io/exclude-pod-placement` or `multiarch.openshift.io/include-pod-placement` labels.
```

Comment:
Uhm, I wouldn't decouple this from the examples for the opt-in/opt-out configurations. I would prefer two sections above stating that:

1. The ClusterPodPlacementConfig can use namespaceSelector to filter (in or out) the namespaces it should process
2. We recommend to use the "opt-out" namespaceSelector, so that the operator can process all the namepsaces, and the users can exclude some by adding the exclusion label (for example,  `multiarch.openshift.io/exclude-pod-placement`).
3. We add another example for the "opt-in" configuration and conclude it by showing that users should set the inclusion label in the namesace they want to be processed by the operator